unit-testing/presentation.html

```diff
@@ -197,18 +278,36 @@ <h4 class="slide-name">TDD VS BDD</h4>
               </tr>
             </tbody>
           </table>
+          <aside class="notes">
```

Comment:
could you remove words BDD and TDD inside td-s?
example:
```
// <p>BDD is a team methodology</p>
// to
<p>team methodology</p>
```

to make slide lightly